de.metas.swat/de.metas.swat.base/src/main/java/de/metas/order/invoicecandidate/C_OrderLine_Handler.java

```diff
@@ -175,8 +175,11 @@ private I_C_Invoice_Candidate createCandidateForOrderLine(final I_C_OrderLine or
 
 		icRecord.setDescription(orderLine.getDescription()); // 03439
 
-		icRecord.setBase_Commission_Points_Per_Price_UOM( orderLine.getBase_Commission_Points_Per_Price_UOM() );
-		icRecord.setTraded_Commission_Percent( orderLine.getTraded_Commission_Percent() );
+		if ( orderLine.getPriceEntered().equals( orderLine.getPriceStd() ) )
```

Comment:
```suggestion
		if ( orderLine.getPriceEntered().compareTo( orderLine.getPriceStd() ) == 0)
```
..we don't want things to happen, just because `1.equals(1.0)` is `false`